standard/sections/clause_7_monitoring_event_topic.adoc

```diff
@@ -46,14 +43,10 @@ The table below provides an overview of the primary topic levels.
 |4
 |centre-id
 |Acronym proposed by Member and endorsed by WMO Secretariat, of the entity producing the event (source)
```

Comment:
|Acronym proposed by Member and endorsed by WMO Secretariat, of the entity subject of the event (target)